service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/history/SealedOrderHistoryServiceTest.java

```diff
@@ -237,11 +244,11 @@ void generateWithPreviousOrdersWithSameApprovalDate() {
         }
 
         @Test
-        void generateWithPreviousLegacyOrdersWithoutApprovalDate() {
+        void generateWithPreviousLegacyOrdersWithoutApprovalDate_WithLinkedApplication() {
```

Comment:
missing the test when the link is empty?

Reply:
All the other scenarios do not have a linked application.